src/main/scala/introcourse/level04/package.scala

```diff
@@ -0,0 +1,15 @@
+package introcourse
+
+package object level04 {
+
+  case class Person(name: String, age: Int)
```

Comment:
let's make case classes `final` 🤓

Reply:
This is one of the good things that we won't teach because we don't need the distraction.

Reply:
I don't think you need to explain anything about it. It's like trying to explain why a class in java can be annotated with `final`. They can look it up or we answer if anyone asks.

Reply:
Additionally, it sets the student in the right path (copy/paste pattern) and the one that knows and understands doesn't see this as miss or anti-pattern.

Having said that the request is irrelevant for this PR and I'm not so fussed about it. So feel free to merge at will.